### PR TITLE
Adds deploy script to bin

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -0,0 +1 @@
+cap production deploy
```
Comment (naps62) [inline bin/deploy]: you're missing `bundle exec`

File: README.md [hunk 2]
```diff
@@ -18,3 +18,11 @@ Here's some of the other things we are using:
 * [Babel](https://babeljs.io/) - A transpiler for ES6
 * [Autoprefixer](https://github.com/middleman/middleman-autoprefixer) -
 Autoprefix CSS
+
+## Deploy
+
+Just run the deploy script:
```
Comment (zamith) [inline README.md]: As per [this](http://www.tameri.com/edit/words2kill.html), let's try to avoid using "just". Ok?

Comment (joaojusto) [inline README.md]: Ok, I guess.

File: bin/deploy (hunk 1 above)
Comment (joaojusto) [inline bin/deploy]: I'll add it

**Pull request merged**